portal/docs/overview/report_payments.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This page looks a lot different in zendesk - it winds up being a bit of text blob without more markup.